src/components/pages/InstructorHome/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import '../../../styles/InstructorStyles/index.less';
 import InstructorSidebar from './InstructorSidebar';
-import InstructorCalender from './InstructorCalender';
+// import InstructorCalender from './InstructorCalender';
```

Comment:
Let's not import this if we're not going to use it. Just remove the commented-out import.